### PR TITLE
[Fix][Bug] Fix docker config naming problem

### PR DESCRIPTION
**Pull request by caicancai**
## What changes were proposed in this pull request

Fix userName to username

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-console/streampark-console-webapp/src/views/setting/system/config.ts [hunk 1]
```diff
@@ -47,12 +47,12 @@ export const settingFormSchema: SettingForm = {
       required: true,
     },
     {
-      field: 'userName',
-      label: t('setting.system.docker.userName.label'),
-      helpMessage: t('setting.system.docker.userName.desc'),
+      field: 'username',
+      label: t('setting.system.docker.username.label'),
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/setting/system/config.ts:51]: Here you need to synchronize the key field in the internationalization file, which is located at streampark-console/streampark-console-webapp/src/locales/lang/en/setting/system.ts (similar for zh_CN).

**Review by wangsizhu0504 — CHANGES_REQUESTED** [review]

File: streampark-console/streampark-console-webapp/src/views/setting/system/config.ts (hunk 1 above)
Comment (caicancai) [reply streampark-console/streampark-console-webapp/src/views/setting/system/config.ts:51]: done

**Review by wangsizhu0504 — COMMENTED** [review]
LGTM!

**Review by wangsizhu0504 — APPROVED** [review]
LGTM！

**Pull request merged**